test_psa_target.py

```diff
@@ -75,8 +74,8 @@ def _build_psa_compliance(args, suite):
 
     retcode = run_cmd_output_realtime(cmd, ROOT)
     if retcode:
-        msg = "Unable to build PSA compliance tests for target - %s" % args.mcu
-        logging.critical(msg)
+        logging.critical(   "Unable to build PSA compliance tests for  \
```

Comment:
Have we considered using `black` to auto format the code? This formatting style looks strange to me.

Reply:
I think its a formatting thing we have been following if the line goes beyond 80 characters, we have to break it, and it also needs to be aligned, I think in the past there were reviews suggested me to correct the alignment. Will start using `black`

Reply:
I have to raise new PR for this if the formatting is to be corrected using `Black`. There will be lot of changes. So this would be for `build_tfm.py`, `build_psa_compliance.py`, `psa_builder.py`, and `test_psa_target.py`. Are we good with that @Patater?

Reply:
I would love to start using `black`. Let's make the switch in a single commit.

Reply:
Do you mean as a new PR with all the files (different commits for each file)?

Reply:
It might be easier to add the `black` changes in a new PR? I think we could just have a single commit for all black changes, as there's nothing to review anyway. `black` won't listen to our review comments 😄